src/Components/Box/index.js

```diff
@@ -7,14 +7,33 @@ import Contribution from '../Contribution';
 
 export default class Box extends Component {
 
+  clickEvent()
+  {
+
+  }
```

Comment:
empty function ?

Reply:
This function is required for on click in phone